Dockerfile

```diff
@@ -1,30 +1,27 @@
- FROM postgres:9.6.1
+FROM mdillon/postgis:9.6-alpine
 
 MAINTAINER David Steinich <drsteini@usgs.gov>
 
 ############################################
-# Install postgis
+# Required for JRE 8 - Java 8 is required to run the Liquibase JAR - lifted from https://github.com/docker-library/openjdk
```

Comment:
I wish Docker supported some way to re-use through composition rather than single inheritance.